src/main/webapp/app/app.constants.js

```diff
@@ -3,10 +3,8 @@
     // DO NOT EDIT THIS FILE, EDIT THE GULP TASK NGCONSTANT SETTINGS INSTEAD WHICH GENERATES THIS FILE
     angular
         .module('artemisApp')
```

Comment:
please revert the commit in this file. It should not be committed to avoid conflicts